packages/frontend/.gitignore

```diff
@@ -32,3 +32,7 @@ yarn-error.log*
 
 # vercel
 .vercel
+
+#Meckis Test files
+components/*Board.tsx
+pages/test.tsx
```

Comment:
This really isn't needed. You should look into how to statically add data to test your comonents or how to use git to only stage certain files

Reply:
isnt that whats .gitignore for?

Reply:
ok, i think im a on a good track here: i did add some static test data and removed the form boilercode entirely. So far so good.

Now i define some static data in my ignored *Board.tsx files, along with with the component that uses the *List.tsx componenents. this *Board component is then used in my test.tsx page. This is the point where i get confused.

I should be able to define some data in the test.tsx file in order to phase out the *Board.tsx files, but then how would i use the *List components? it would also still leave the test.tsx file in place and clutter the codebase. My best guess would be to start the real page for them in /pages, relieving test.tsx from its reason to exist and which would pretty much come full circle to me having it in the gutted and ignored *Board.tsx files with the only difference beeing it cluttering the codebase again. so, i cant really wrap my head around how else i should test my components as long as they are still unfinished.

![videoboard](https://user-images.githubusercontent.com/85892774/122630494-83e62e80-d0c4-11eb-840c-20ab9b50cae1.png)

and somehow this does seem to be a good start for the actual video page, it is a lot less cluttered then the version with with the form boilerplate

Help on that matter would be very much appreciated

Reply:
So if my understanding is correct, you want to move the static data from your `Board.tsx` component to `test.tsx` page right ? so you can remove the `Board.tsx`

Reply:
that is exactly the question, i dont really know how to solve the issue with the gitignore file. I could create the actual video page, but where would i then place the static data in a way that it is not cluttering the codebase. I could write a seperat file with the data, but that would be overkill and also clutter the codebase except when i add them to gitignore.

i dont know how i can solve reds request to remove the lines he mentions

i guess it is a problem with me not properly understanding how git is supposed to work or what gitignore is supposed to be

Reply:
do you wish to push the static data to the repo ? if yes I suggest you to store the static data in the same file as your component
and actually you dont have to include your test files
```
#Meckis Test files (Temporary solution)
components/Video/VideoBoard.tsx
components/Image/ImageBoard.tsx
pages/test.tsx
```
in the .gitignore, you can just exclude them when you want to create a commit

Reply:
i do not wish to push my testing data as i think that i would be unnecessary overhead. 
yeah i know that i can just dont stage them but i was hoping for a nice and automated way that makes my IDE happy lmao, i will do that from now on

Reply:
Usually you just wouldn't need the static data after you're done implementing, thats why I'm confused as to why you need those test files. Then again maybe this is a call to implement storybook, ill look into if that makes any sense later.

Reply:
yeah exactly, why would i need static data in the finished component. it is not finished tho, the purpose of this PR has changed drastically, it is basically trying to finish up Video, so i think that i should implement the components in the `/clips` page and finish up responsiveness for the player before this pr is ready to be merged

Reply:
figured out how to do the responsive player, i should now be ready to merge